packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceSourceDiff.test.ts

```diff
@@ -153,23 +153,6 @@ describe('Force Source Diff', () => {
       );
     });
 
-    it('Should show message when remote file is not found in org', async () => {
```

Comment:
moving this test to "Should throw error for empty cache" since the error is caught in the commandlet.